trio/testing/_fake_net.py

```diff
@@ -299,6 +300,10 @@ async def sendmsg(self, *args: Any) -> int:
         ancdata = []
         flags = 0
         address = None
+
+        # This does *not* match up with socket.socket.sendmsg (!!!)
+        # https://docs.python.org/3/library/socket.html#socket.socket.sendmsg
+        # they always have (buffers, ancdata, flags, address)
```

Comment:
I started writing coverage tests for `_fake_net` which is how I noticed this, but leaving those for another PR.